### PR TITLE
Add Clover Configurator.app v4.41.1

### PR DESCRIPTION
**Pull request by vin047**
Add Clover Configurator.app v4.41.1 to homebrew cask

*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

### CODE REVIEW

File: Casks/clover-configurator.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'clover-configurator' do
+  version '4.41.1'
+  sha256 'b766115c850e0d3b05d525ec4a8b13938693624d6bd5eb16c1f82ef16e923a8f'
```
Comment (numbermaniac) [inline Casks/clover-configurator.rb]: The download link appears to be versionless, so you may need to use `version :latest` and `sha256 :no_check`.

**Comment by miccal** [comment]
Verified locally.

**Comment by numbermaniac** [comment]
@miccal This is an unversioned URL - isn't the standard to usually have `sha256 :no_check` and `version :latest`?

**Comment by miccal** [comment]
@numbermaniac but it has an `appcast`, meaning it also requires a `version` and `sha256`.

**Pull request merged**